_memo_keys/ICS20_memo_keys.json

```diff
@@ -0,0 +1,62 @@
+{
+  "memo_keys": [
+    {
+      "key": "wasm",
+      "description": "To call a cosmwasm contract",
+      "git_repo": "https://github.com/osmosis-labs/osmosis/tree/main/x/ibc-hooks#ibc-hooks",
+      "memo": {
+        "wasm": {
+          "contract": "osmo1contractAddr",
+          "msg": {
+            "raw_message_fields": "raw_message_data"
+          }
+        }
+      }
+    },
+    {
+      "key": "ibc_callback",
+      "description": "Passing the ibc acknowledgement back to a cosmwasm smart contract",
+      "git_repo": "https://github.com/osmosis-labs/osmosis/tree/main/x/ibc-hooks#ibc-hooks",
+      "memo": {
+        "ibc_callback": "osmo1contractAddr"
+      }
+    },
+    {
+      "key": "autopilot",
```

Comment:
`autopilot` can support multiple message types https://github.com/Stride-Labs/stride/tree/main/x/autopilot

For example, 1-click liquid stake
```
{ 
    "autopilot": {
          "receiver": "strideXXX", 
          "stakeibc": {
               "stride_address": "strideXXX",
               "action": "LiquidStake",
          }
    }
}
```

and Update Airdrop Address
```
{ 
    "autopilot": {
          "receiver": "strideXXX", 
          "claim": {
               "stride_address": "strideXXX",
               "airdrop_id": "evmos",
          }
    }
}
```

Can we include both here?

Reply:
just thinking out loud--The key objects have been set to 'additionalProperties: true', so we might be able to do something like 'memo' and 'memo2'. (no schema change required).
Or we could change memo to an array, 'memos', and then we can have one or multiple memo structures associated with a single key (schema changes required). @womensrights wyt?